Frameworks/Foundation/NSCFArray.mm

```diff
@@ -86,7 +82,8 @@ - (NSUInteger)count {
 
 - (id)objectAtIndex:(NSUInteger)index {
     if (index >= CFArrayGetCount((CFArrayRef)self)) {
-        [NSException raise:@"Array out of bounds" format:@"objectAtIndex: index > count (%d > %d), throwing exception\n", index, CFArrayGetCount((CFArrayRef)self)];
+        [NSException raise:@"Array out of bounds"
```

Comment:
While you're reformatting this, can you remove the text from the exception that says it's throwing an exception? that's just silly!